src/components/TransactionsList.js

```diff
@@ -1,38 +1,51 @@
 import React from 'react'
+import { Link, Route } from 'react-router-dom'
+import Transaction from './Transaction'
 
-const TransactionsList = () => {
+export default function TransactionsList(props) {
 
   return (
-    <table className="ui celled striped padded table">
-      <tbody>
-        <tr>
-          <th>
-            <h3 className="ui center aligned header">
-              Posted At
-            </h3>
-          </th>
-          <th>
-            <h3 className="ui center aligned header">
-              Description
-            </h3>
-          </th>
-          <th>
-            <h3 className="ui center aligned header">
-              Category
-            </h3>
-          </th>
-          <th>
-            <h3 className="ui center aligned header">
-              Amount
-            </h3>
-          </th>
-        </tr>
+    <div>
+      <p></p>
+      <h2>Transaction List</h2>
+      <table id="transactions-list" className="ui celled striped padded table">
+        <tbody>
+          <tr>
+            <th>
+              <h3 className="ui center aligned header">Posted At</h3>
+            </th>
+            <th>
+              <h3 className="ui center aligned header">Description</h3>
+            </th>
+            <th>
+              <h3 className="ui center aligned header">Category</h3>
+            </th>
+            <th>
+              <h3 className="ui center aligned header">Amount</h3>
+            </th>
+          </tr>
 
-        {"... your code here..."}
+          {props.transactions.map( transaction => {
 
-      </tbody>
-    </table>
+               if (transaction.description.toLowerCase().match(props.searchTerm.toLowerCase()) ||
```

Comment:
I would consider doing this filtering on the `Container` component to keep this component a bit simpler - just a thought.